blocks/header/header.html

```diff
@@ -0,0 +1,90 @@
+            </svg>   
+        </a>
+        <div class="header-top-right">
+            <a target="_blank" href="/subscribe-golf-digest-plus" class="header-Subscribe header-link">Subscribe</a>
+            <div class="profileInfo">
+                <a data-logged-in="false" class="profileInfo-ProfilePhoto" id="nav-photo" href="//www.golfdigest.com/my-account">
+                </a>
+                <a class="profileInfo-Button--Login" href="#" data-type="gigya-login">Log In</a>
+            </div>
+            <button class="header-search-button" data-type="button-search-toggle" aria-label="Search Toggle">
+                <svg width="25" height="25" viewBox="0 0 25 25" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Search Icon">
+                    <path d="M10.7954 3.125C9.27835 3.125 7.79535 3.57486 6.53396 4.4177C5.27257 5.26053 4.28943 6.45849 3.70888 7.86007C3.12832 9.26166 2.97642 10.8039 3.27239 12.2918C3.56835 13.7797 4.29889 15.1465 5.37161 16.2192C6.44434 17.2919 7.81108 18.0225 9.29899 18.3184C10.7869 18.6144 12.3292 18.4625 13.7308 17.8819C15.1323 17.3014 16.3303 16.3183 17.1731 15.0569C18.016 13.7955 18.4658 12.3125 18.4658 10.7954C18.4657 8.76113 17.6575 6.81021 16.2191 5.37175C14.7806 3.9333 12.8297 3.12513 10.7954 3.125V3.125Z" stroke="black" stroke-width="2" stroke-miterlimit="10"></path>
+                    <path d="M16.5181 16.5181L21.875 21.875" stroke="black" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round"></path>
+                </svg>
+            </button>
+        </div>
+    </div>
+    <hr>
+    <div class="header-bottom-bar no-children">
+        <!-- Bar -->    
+        <a class="header-sub-channel o-Header__a-Link" href="//www.golfdigest.com/the-loop">
+        The Loop
+        </a>
+        <div class="separator"></div>
+        <div class="header-SubNav"></div>
+        <div class="header-bottom-right">
+            <form action="/search" method="get" data-module="search-form" class="search-form" data-mobile-search-box="" id="mod-search-form-1">
+                <script type="text/x-config">
```

Comment:
I think you can remove that and put it into the header.js file

Reply:
@icaraps Can you clarify this? You previously recommended putting header html template into a separate file. It also contradicts your last comment in this PR.

Reply:
@hsaginor ha sorry for the confusion... I meant specifically here the <script> tag unless you actually are needing it in the html file for some reason ?

Reply:
@icaraps oh I think that's just part of Gigya integration which was copied along along with some of the original markup. It's not working right now and we will need to revisit this at some point. We can probably just remove it for now. 

That being said this is a JSON configuration object I believe Gygia API would be looking for in the markup. So, it probably doesn't belong with header block's JS.